async-nats/Cargo.toml

```diff
@@ -34,6 +34,7 @@ webpki-roots = "0.22"
 nuid = "0.3.2"
 serde_nanos = "0.1.1"
 time = { version = "0.3.6", features = ["parsing", "formatting", "serde", "serde-well-known"] }
+secstr = "0.3.1"
```

Comment:
I don't see much value in using `secstr`. 
We decided to not use it, at least for now.

Reply:
Ok! I'll remove this.